portal/src/components/Typography/Typography.tsx

```diff
@@ -17,11 +17,14 @@ export const HeadingLarge: React.FC = ({ children, ...rest }) => {
     );
 };
 
-export const HeadingMedium: React.FC = ({ children, ...rest }) => (
-    <h3 className="jkl-heading-medium jkl-portal-heading-medium" {...rest}>
-        {children}
-    </h3>
-);
+export const HeadingMedium: React.FC = ({ children, ...rest }) => {
+    const id = typeof children === "string" ? children.toLowerCase().replace(/[^\wæøåÆØÅ]+/g, "-") : undefined;
```

Comment:
w?

Reply:
Du kan bruke `React.Children.only(children)` hvis du vil kreve at det er et child, den vil kaste feil hvis man får inn to children. Evt `React.Children.count(children) === 1`

Reply:
Ja det høres lurt ut. Så ikke så gæli mye på det, bare ren copy paste fra <h2>. Ser på det begge steder.

Reply:
Tatt en titt, virker som om det som er der i dag er en like god løsning, men forsto ikke forslagene 100% så mulig jeg misforstår. Hva er det som er ugreit?

Reply:
Kommenterer her siden utgangspunktet er min kode 😅

@lfbergee Meningen her er ikke å sjekke _om_ det finnes `children`, men å generere en `id` til overskriften _ut fra_ `children`, dersom `children` er en tekststreng 😊 Litt usikker på hvordan `React.Children` vil utgjøre noen forskjell for det caset.

Det man nok uansett burde gjøre er å trekke ut formateringen til en `hyphenate()`-funksjon e.l. Det burde også hjelpe på lesbarhet 😊

Reply:
fair enough